srcpkgs/gitea/template

```diff
@@ -32,7 +32,7 @@ license="MIT"
 homepage="https://gitea.io"
 changelog="https://github.com/go-gitea/gitea/blob/master/CHANGELOG.md"
```

Comment:
This should link to a raw file.

Reply:
https://raw.githubusercontent.com/go-gitea/gitea/main/CHANGELOG.md